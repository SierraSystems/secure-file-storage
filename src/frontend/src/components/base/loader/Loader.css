.page-loader {
  height: 60px;
  width: 60px;
  color: #494949;
  border: 7px solid;
  border-radius: 50%;
  border-top-color: transparent;
  animation: rotate 2s linear infinite;
  display: inline-block;
}

@keyframes rotate {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
