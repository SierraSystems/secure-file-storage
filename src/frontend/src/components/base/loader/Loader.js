import React from "react";
import "./Loader.css";

export default function Loader() {
  return (
    <div className="loader-align">
      <div className="page-loader" />
      <br />
      Loading... Please Wait
    </div>
  );
}
