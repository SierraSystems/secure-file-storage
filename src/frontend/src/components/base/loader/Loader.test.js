import React from "react";
import Loader from "./Loader";
import testBasicSnapshot from "../../../TestHelper";

describe("Loader", () => {
  test("Component renders as expected", () => {
    testBasicSnapshot(<Loader />);
  });
});
