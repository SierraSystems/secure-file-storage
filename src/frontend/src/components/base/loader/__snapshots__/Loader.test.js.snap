// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component renders as expected 1`] = `
<div
  className="loader-align"
>
  <div
    className="page-loader"
  />
  <br />
  Loading... Please Wait
</div>
`;
